swift/hummingbird-framework/Package.swift

```diff
@@ -0,0 +1,27 @@
+// swift-tools-version:5.3
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "server",
+    products: [
+        .executable(name: "server", targets: ["server"])
+    ],
+    dependencies: [
+        .package(url: "https://github.com/hummingbird-project/hummingbird.git", .upToNextMinor(from: "0.6.0")),
+    ],
+    targets: [
+        .target(name: "server",
+            dependencies: [
+                .product(name: "Hummingbird", package: "hummingbird"),
+            ],
+            swiftSettings: [
```

Comment:
this is awesome 🎉 

perhaps a good idea to use this for all `swift` **frameworks** @0xTim

Reply:
Yes we should probably add it to all Swift frameworks that work with it